js/components/foia_submission_address.jsx

```diff
@@ -12,12 +12,18 @@ function FoiaSubmissionAddress({ submissionAddress }) {
     postal_code,
   } = submissionAddress;
 
+  // Address line seems to have been parsed wrong, it seems to be: name,
+  // title, department, suite. And then address_line2 is actually line1.
+  const [name, title, department, suite] = address_line1.split(',', 4);
+
   return (
     <address className="submission-help_mailing">
-      <p>{ additional_name }</p>
-      <p>{ address_line1 }</p>
-      <p>{ address_line2 }</p>
-      <p>{ `${locality}, ${administrative_area} ${postal_code}` }</p>
+      <div>{ additional_name }</div>
```

Comment:
So, actually, I think within `address` we don't need additional tags -- [MDN just uses `br`s.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)

Reply:
hmm, this is actually really hard. JSX prefers things to be wrapped in a single container, and there's no way to do something like { address_line1 && {address_line1}<br/> } You'd still have to wrap it in a span or something.

Reply:
even something like `[name, line1, line2,].join(<br />)` doesn't work.